src/complex/Utilities/DataArrayUtilities.hpp

```diff
@@ -178,38 +191,36 @@ void ReplaceValue(DataArray<T>& inputArrayPtr, const DataArray<ConditionalType>*
  * return FALSE if the wrong array type is specified as the template parameter
  */
 template <class T>
-bool ConditionalReplaceValueInArrayFromString(const std::string& valueAsStr, DataObject& inputDataObject, const IDataArray& conditionalDataArray)
+std::pair<std::string, int32> ConditionalReplaceValueInArrayFromString(const std::string& valueAsStr, DataObject& inputDataObject, const IDataArray& conditionalDataArray)
```

Comment:
ConditionalReplaceValueInArrayFromString should return a Result<>.

Reply:
Done